src/database.ts

```diff
@@ -98,10 +98,17 @@ export class Database<D = unknown> {
   ): Promise<Array<Result<T>>> {
     try {
       const start = performance.now();
+      const nameMap =
+        typeof this.config.aliases?.read === "function"
+          ? await this.config.aliases?.read()
```

Comment:
I was doing the read each time in case there are two `Database` instances making writes. In which case you would need to have some kind of way to avoid stale caches.
The interface might be a little confusing, but if the dev wanted to optimize by putting the aliases in memory I was thinking they would do something like:
```
const nameMap = {}; // they could always do an async "fetch" here to get existing aliases
const db = new Database({
  signer,
  aliases: {
    read: async function () {
      return nameMap;
    },
    write: async function (names) {
      for (const uuTableName in names) {
        nameMap[uuTableName] = names[uuTableName];
      }
    },
  },
});
```

They would also need to build some logic to store the aliases permanently.
Just to brainstorm...
Would it be useful to enable another helper that managed an in memory Object somehow?
Maybe something like:
```
import { Database, helpers } from "@tableland/sdk":

// get the existing state however the dev wants
const aliasesObj = await doFetchToGetAliases();
const memAliases = new helpers.MemoryAliases(aliasesObj);

const db = new Database({
  signer,
  aliases: memAliases,
});

// ...later on, whenever it makes sense...
const newMap = memAliases.read();
await doPostToUpdateAliases(newMap);
```